x-pack/filebeat/input/s3/config.go

```diff
@@ -42,16 +44,22 @@ func (c *config) Validate() error {
 		return fmt.Errorf("visibility timeout %v is not within the "+
 			"required range 0s to 12h", c.VisibilityTimeout)
 	}
+
 	if c.APITimeout < 0 || c.APITimeout > c.VisibilityTimeout/2 {
 		return fmt.Errorf("api timeout %v needs to be larger than"+
 			" 0s and smaller than half of the visibility timeout", c.APITimeout)
 	}
+
```

Comment:
this one too?

Reply:
yes :) I just want to make it look prettier 😂

Reply:
👍